CONTRIBUTORS.md

```diff
@@ -74,3 +74,4 @@ and we will add you. **All** contributors belong here. 💯
 * [Kevin Barbour](https://github.com/kevinbarbour)
 * [Epsxy](https://github.com/epsxy)
 * [Jens Arnfast](https://github.com/jarnfast)
+* [Madhu M Pandurangi](https://github.com/MadhuMPandurangi)
```

Comment:
🎉

Reply:
Thanks for adding. I'll look into the edits that need to be done in this.